.travis.yml

```diff
@@ -2,13 +2,13 @@ language: python
 
 matrix:
   include:
-    - python: 2.7
-      env: TOXENV=py27
-    - python: 2.7
+    - python: 3.5
+      env: TOXENV=py35
```

Comment:
Shouldnt we keep running the tests in 2.7 (this in the lint one ?) as long as we still release a version in 2.7 ?